dir.props

```diff
@@ -153,7 +153,17 @@
 
     <!-- PreReleaseSuffix for TFS win8-arm CoreCLR package -->
     <ExternalExpectedPrerelease>servicing-24607-00</ExternalExpectedPrerelease>
- 
+
+    <!-- Central place to set the versions of all nuget packages produced in the repo -->
+    <Version Condition="'$(Version)' == ''">1.0.5</Version>
```

Comment:
Might want to do the same conditions as in 1.1 `'$(MSBuildProjectExtension)' == '.pkgproj'`